src/main/java/service/report/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.report;
+
+import dao.Storage;
+import model.FruitRecord;
+
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String FRUIT = "fruit";
+    private static final String QUANTITY = "quantity";
+    private static final String COMMA = ",";
+
+    @Override
+    public String getReport(List<FruitRecord> fruitRecords) {
+        StringBuilder report = new StringBuilder();
+        report.append(FRUIT).append(COMMA).append(QUANTITY).append(System.lineSeparator());
+
+        Storage.storage.forEach((fruit, quantity) -> report
+                .append(fruit).append(COMMA).append(quantity).append(System.lineSeparator()));
+        return report.toString();
+    }
+}
```

Comment:
The implementation of `ReportService` should not directly use the `Storage` class. This violates the Single Responsibility Principle (SRP) and the Dependency Inversion Principle (DIP) from SOLID. The service should receive the necessary data as a parameter rather than reaching out to `Storage` directly. Consider refactoring the code to pass the needed data to the `getReport` method.